### ISSUE TITLE
[MU4 Issue] Drumset not yet mapped in Muse Sounds

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#14016]
**Describe the bug**
The Drumset instrument does not seem to map to anything in Muse Sounds. 

**To Reproduce**
1. Create a new score with the Drumset instrument
2. Activate the Muse Sounds profile in View --> Playback setup
3. Observe in the Mixer: Sound that MS Basic is still selected
4. Also observe that there is no instrument available called Drumset (Or Drum Kit) in the Muse Sounds mixer menu

**Expected behavior**
MS Basic contains 25 different instrument sounds within the Drumset instrument. Where these are available in Muse Sounds, they should be mapped accordingly (see relevant noteheads below):
<img width="254" alt="Screenshot 2022-10-25 at 3 32 09 pm" src="https://user-images.githubusercontent.com/86290556/197787250-2314cc5b-d7a5-4e64-8c3f-e480318b30e6.png">

<img width="2297" alt="Screenshot 2022-10-25 at 4 03 08 pm" src="https://user-images.githubusercontent.com/86290556/197796863-5ee68d64-c193-4f4e-9a4c-700137c69265.png">

**Platform information**
 - MacOS 12.6 (M1) 

@Tantacrul FYI

**Comment by erinic04** [comment]
It would also be nice to have one of these for an orchestral percussion staff so we don't have to split the percussion staves per sound

**Comment by vpereverzev** [comment]
I suggest to keep a single issue with MuseSounds mappings, otherwise it'd be hard to collect scattered info

**Comment by Tantacrul** [comment]
I think the drumset is unique though. We need to agree on mappings before plugging in the new drumkit sounds. Any other 'missing instrument' mapping should be kept together though. I agree with that. I have one document outlining all the incorrect mappings so far, so I can keep track.

**Comment by MarcSabatella** [comment]
I would agree the “orchestral percussion” instrument mapping is way more important t than the “drumset” instrument.  The latter really isn’t a good fit for Muse Sounds even if the available sounds map decently.

**Comment by bkunda** [comment]
We are going to postpone this for now because we don't have all the available sounds we need for the drum kit in Muse Sounds.

**Comment by erinic04** [comment]
@bkunda are there any plans to do a general "orchestral percussion" sound set for combined staves?

**Comment by bkunda** [comment]
@erinic04 Unfortunately I don't think this is going to be possible for the 4.0 release because we don't yet have the full complement of instrument sounds in Muse Sounds. But this is certainly something we'll want to tackle as soon as we can :-)

**Comment by erinic04** [comment]
Cool thanks :)

**Comment by GrahamDickins** [comment]
Hi there, now 4.0 is released, is this still a planned feature? I currently have a score for drum kit that has to be on MS Basic as there's no combined Muse Sounds equivalent (just individual sounds for kick, snare etc under percussion). Would love for it to have the full 4.0 sound without splitting each drum into separate parts. Thanks, Graham

**Comment by cbjeukendrup** [comment]
Yes, this is still going to happen! Not sure yet when, though.

**Comment by henkdegroot** [comment]
I was trying to use the Drum Kit (which is available now in MuseSounds) in a score and found that I cannot alter the placement of the notes without this effecting the sound being played. This happens when using the Drum Kit available in the CinePerc samples.

For instance, I don't want the hi-hat to be placed above the staff but instead I would like it to show between the two top staff lines. After altering the Drum Set definition, the notes are placed as desired but the sound has changed from hi-hat to bass drum.

Not sure where I should report this issue and found this one on github which related to mapping drumset, hence I added my comment here. If a separate issue for this is preferred, let me know.

FYI, I am using MS 4.3.2 + MuseHub 2.0.15.1305.

**Comment by shoogle** [comment]
@henkdegroot, please describe the exact process you used. Were there notes in the score before you changed sound in the Mixer? Did you change notation via the Edit Drumset dialog? Did you edit the appearance an existing pitch in that dialog (which pitch was it?) or did you create a new pitch and define notation for it?

**Comment by henkdegroot** [comment]
Here are steps:
1. Start a new score and select Large Drum Kit in the instrument list
2. Open the Mixer, this shows the current assigned sound is MS Basic, change this to Muse Sounds->Muse->Muse Percussion->Drum Kit
3. Start entering notes (press N), at this moment the drumset (shown at the bottom of the dialog) shows 13 icons (a lot less compared to the MS Basic Drum Kit but could be due to different sounds being availbale)
4. Enter a couple of Hi-Hat notes (press shortcut G a few times) and exit note entry
5. During the entry and when selecting a note in the score the hi-hat sounds is played
6. Right click the staff and select Edit drumset
7. The Edit drumset dialog shows the Hi-Hat at No. 42, Note f#2
8. Change the notehead group from Cross to Normal and click OK
9. The notehead changed in the score. Now click one of the notes and the Bass Drum sound is played.

Obvious it is not common to use a normal notehead for a hi-hat but when I enter a drumpart from an existsing paperscore, I normally like to keep the same notation as the original. At first I thought moving the note to a different staff line caused the sound to change, later I discovered it was the notehead changed which caused this.

**Comment by zacjansheski** [comment]
Closing as resolved by the latest Muse Percussion update
